src/autopas/molecularDynamics/LJFunctorAVX.h

```diff
@@ -436,14 +480,11 @@ class LJFunctorAVX
       const __m128d hSumfxfyHigh = _mm256_extractf128_pd(hSumfxfy, 1);
       const __m128d hSumfzHigh = _mm256_extractf128_pd(hSumfz, 1);
 
-      const union {
-        __m128d reg;
-        double arr[2];
-      } sumfxfyVEC = {.reg = _mm_add_pd(hSumfxfyLow, hSumfxfyHigh)};
+      const __m128d sumfxfyVEC = _mm_add_pd(hSumfxfyLow, hSumfxfyHigh);
```

Comment:
what exactly was this union workaround and why/since when is it not needed anymore?